main.tf

```diff
@@ -2,6 +2,11 @@ data "azurerm_resource_group" "main" {
   name = var.resource_group_name
 }
 
+data "azurerm_kubernetes_service_versions" "current" {
+  location        = data.azurerm_resource_group.main.location
+  include_preview = false
+}
+
```

Comment:
Since no version specified, `azurerm_kubernetes_cluster` will use the latest recommended version. Could we remove this to simplify the code?

Reply:
Absolutely! Great point 👍🏼